rasa/nlu/model.py

```diff
@@ -289,12 +289,11 @@ def ensure_model_compatibility(
         model_version = metadata.get("rasa_version", "0.0.0")
         if version.parse(model_version) < version.parse(version_to_check):
             raise UnsupportedModelError(
-                "The model version is too old to be "
-                "loaded by this Rasa NLU instance. "
-                "Either retrain the model, or run with "
-                "an older version. "
-                "Model version: {} Instance version: {}"
-                "".format(model_version, rasa.__version__)
+                f"The model version is trained using Rasa Open Source {model_version} "
+                f"and is not compatible with your current installation ({rasa.__version__}). "
+                "This means that you either need to retrain your model "
+                "or revert back to the Rasa version that trained the model "
+                "to ensure that the versions match up again."
```

Comment:
I think this is somewhere in our code engineering guidelines. If one part of a string uses an f-string, the other parts should also use it for consistency.
```suggestion
                f"The model version is trained using Rasa Open Source {model_version} "
                f"and is not compatible with your current installation ({rasa.__version__}). "
                f"This means that you either need to retrain your model "
                f"or revert back to the Rasa version that trained the model "
                f"to ensure that the versions match up again."
```